sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -142,6 +147,7 @@ export class KeysClient {
       );
     }
     requestPolicyFactories = requestPolicyFactories.concat([
+      tracingPolicy(),
```

Comment:
Apologies if obvious question - but can this capture the user-facing SDK method name in the trace? I don't quite see how this is tied to that.

Reply:
It can't right now, so always uses the name `core-http`.  In order to support capturing the user-facing SDK method name (well, really the method name of the low-level API), we would need to add the operation name on the `WebResource`.

My vote would be to add an `operationName` to the `OperationSpec` interface, which `WebResource` already has access to:
https://github.com/Azure/azure-sdk-for-js/blob/c71fe51a8e5beb6afb7fecf013c5ace605d3e77c/sdk/core/core-http/lib/operationSpec.ts#L12

But there's not currently a way to get the name from a policy.

Reply:
Discussed internally but reiterating here. The tracing policy is meant to capture the http call, so we'll still need what we have in keyvaults today where we create an operation-specific span for each exposed method. We do intend to have core-http spans be children of the operation-specific spans.